packages/components/psammead-most-read/src/Item/index.jsx

```diff
@@ -102,11 +111,15 @@ MostReadLink.propTypes = {
   title: string.isRequired,
   href: string.isRequired,
   children: node, // this node will be a timestamp container
+  typography: oneOf(['greatprimer', 'pica']),
+  rankTypography: oneOf(['foolscap', 'trafalgar']),
```

Comment:
If you only want to allow these values, you should also update the documentation props table with a description to ensure that's understood.